docs/jobs/openings/frontend.md

```diff
@@ -26,8 +22,8 @@ We are looking for a frontend and web3 wizard to work with us on the [Aragon dap
 - Write an email with the following attachments plus any information you consider important to *hiring@aragon.one*, with the subject `Frontend Dev opening application`.
 
 - Please send along your GitHub profile. Since we are open source only, we require applicants for developer positions to have a very active and healthy GitHub profile.
-
-- Please send the following challenge completed.
+- Bonus: Why did Aragon caught your attention? Which challenge within the crypto space are you more interested in?
```

Comment:
Same as in dev_rel.md, should probably be `most interested in` or `more interested in than others`?

Reply:
Same as dev_rel.md, since it's the same let's keep it short.